guide/english/computer-science/threads/index.md

```diff
@@ -32,6 +32,35 @@ if __name__ == "__main__":
     for i in range(1, 11):
         threading.Thread(target=print_number, args=(i,)).start()
 ````
+Java Implementation
+````
```

Comment:
add java at the end of ```` which will provide syntax highlighting to the code block you have inserted, so
instead of having
````
class print_thread...
````

you can write:

````java
class print_thread ...
````